presto-cli/src/main/java/io/prestosql/cli/InputHighlighter.java

```diff
@@ -88,6 +89,12 @@ else if (isComment(type)) {
         return builder.toAttributedString();
     }
 
+    @Override
+    public void setErrorPattern(Pattern pattern) {}
```

Comment:
do you know who calls this and whether it's fully safe to ignore this?

Reply:
Nope. I was trying to understand that but no luck. I've seen in the `DefaultHighlighter` that `errorPattern` and index are used directly by the `highlight` method to change style when `buffer` matches. I decided that it's ok to omit that functionality for now.

Reply:
i saw that too. Thanks for checking.